### ISSUE TITLE
[Label] Accessibility violation when Upload/FileSelect is associated with KendoLabel

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#3022]
**Describe the bug**
When an Upload or FileSelect component is associated with a KendoLabel, the inner Input element does not receive an aria-labelledby attribute (as it goes on its focusable wrapper). Screen readers are working as expected, but the AXE accessibility tool throws the following warning:

Issue description
Ensures every form element has a label

<input kendofileselect="" ng-reflect-dir="ltr" ng-reflect-disabled="false" ng-reflect-multiple="true" ng-reflect-restrictions="[object Object]" type="file" autocomplete="off" tabindex="-1" name="files" multiple="multiple" dir="ltr">

Fix at least one (1) of these issues:
aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Form element does not have an implicit (wrapped) <label>

Form element does not have an explicit <label>

Element has no title attribute or the title attribute is empty

**Expected behavior**
WCAG: 1.3.1, 4.1.2 criteria are met when the Upload/FileSelect is associated with a KendoLabel.

**Comment by preslavapetrova1** [comment]
Fixed in `v7.1.0` of the `kendo-angular-upload` package
